#This message is used to send command to tetra
#Angular Position Mode command

int32 angular_degree
---
bool command_Result

